commander/src/utils/node/commons.ts

```diff
@@ -158,3 +164,24 @@ export const validateVersion = async (
 		throw new Error(error.message);
 	}
 };
+
+export const getSemver = (str: string): string => {
```

Comment:
no, it only extracts if you give something like this `semver.coerce('dummy 42.6.7.9.3-alpha dummy')` for us, we need to extract the version from URL, so it can't parse it.

Reply:
it will not work for this `semver.coerce('https://lisk-releases.ams3.digitaloceanspaces.com/lisk-core/lisk-1.1.0-5556904-Linux-x86_64.tar.gz')`